monocle/db.py

```diff
@@ -310,6 +354,28 @@ class FortSighting(Base):
     )
 
 
+class RaidSighting(Base):
+    __tablename__ = 'fort_raids'
+    id = Column(Integer, primary_key=True)
+    fort_id = Column(Integer, ForeignKey('forts.id'))
+    raid_battle_ms = Column(Integer, index=True)
+    raid_spawn_ms = Column(Integer, index=True)
+    raid_end_ms = Column(Integer, index=True)
+    raid_level = Column(Integer)
+    complete = Column(TINY_TYPE)
+    pokemon_id = Column(TINY_TYPE)
```

Comment:
Getting error in scan.log when pokemon_id is ''. Perhaps when a raid is detected, but the pokemon_id isn't revealed yet.

sqlalchemy.exc.OperationalError: (raised as a result of Query-invoked autoflush; consider using a session.no_autoflush block if this flush is occurring prematurely) (_mysql_exceptions.OperationalError) (1366, "Incorrect integer value: '' for column 'pokemon_id' at row 1") [SQL: 'INSERT INTO fort_raids (fort_id, raid_battle_ms, raid_spawn_ms, raid_end_ms, raid_level, complete, pokemon_id, cp, move_1, move_2) VALUES (%s, %s, %s, %s, %s, %s, %s, %s, %s, %s)'] [parameters: (25, 1498426807, 1498419607, 1498430407, 1, False, '', '', '', '')]